core/kazoo_documents-1.0.0/src/kzd_alert.erl

```diff
@@ -0,0 +1,254 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2015, 2600Hz
+%%% @doc
+%%% Alert document
+%%% @end
+%%% @contributors
+%%%   Peter Defebvre
+%%%-------------------------------------------------------------------
+-module(kzd_alert).
+
+-export([new/0
+         ,type/0
+         ,id/1
+         ,fetch/1
+        ]).
+
+-export([title/0, title/1, title/2, set_title/2
+         ,category/0 ,category/1, category/2, set_category/2
+         ,message/0, message/1, message/2, set_message/2
+         ,metadata/0, metadata/1, metadata/2, set_metadata/2
+         ,level/0, level/1, level/2, set_level/2
+         ,from/0, from/1, from/2, set_from/2
+         ,to/0, to/1, to/2, set_to/2
+         ,expiration_date/0, expiration_date/1, expiration_date/2, set_expiration_date/2
+         ,expired/1
+        ]).
+
+-define(ID, <<"_id">>).
+-define(PVT_TYPE, <<"alert">>).
+
+-define(TITLE, <<"title">>).
+-define(CATEGORY, <<"category">>).
+-define(MESSAGE, <<"message">>).
+-define(METADATA, <<"metadata">>).
+-define(LEVEL, <<"level">>).
+-define(FROM, <<"from">>).
+-define(TO, <<"to">>).
+-define(EXPIRATION_DATE, <<"expiration_date">>).
+
+-include("kz_documents.hrl").
+
+-type doc() :: wh_json:object().
+-export_type([doc/0]).
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+-spec new() -> doc().
+new() ->
+    wh_doc:set_type(wh_json:new(), ?PVT_TYPE).
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+-spec type() -> ne_binary().
+type() -> ?PVT_TYPE.
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+-spec id(doc()) -> api_binary().
+id(JObj) ->
+    wh_doc:id(JObj).
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+-spec fetch(api_binary()) -> {'ok', doc()} |
+                            {'error', any()}.
+fetch('undefined') ->
+    {'error', 'invalid_db_name'};
+fetch(<<_/binary>> = AlertId) ->
+    couch_mgr:open_cache_doc(?WH_ALERTS_DB, AlertId).
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+-spec title() -> ne_binary().
+-spec title(doc()) -> api_binary().
```

Comment:
I agree and I asked multiple time for these commands. Just keep me in the loop on these tools so that I do not waste anyone's time.